IDE/Renesas/e2studio/RZN2L/test/src/wolfSSL/.gitignore

```diff
@@ -0,0 +1,2 @@
+*
+!.gitignore
```

Comment:
Do we need this `wolfSSL` directory? Seems like it is empty apart from this `.gitignore`.

Reply:
The folder is a place folder for e2studio project. In the folder, there are  source files which are linked to ssl.c, tls.c and so on in wolfssl source tree  when opening the project file.